src/main/java/se/sundsvall/supportmanagement/api/model/communication/CommunicationAttachment.java

```diff
@@ -0,0 +1,98 @@
+package se.sundsvall.supportmanagement.api.model.communication;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.Objects;
+
+import jakarta.validation.constraints.NotBlank;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class CommunicationAttachment {
+
+	@NotBlank
```

Comment:
I dont think validation is needed/used as the class is only used in response from service (comment is applicable for all places where validation annotations are used in class)?